rocAL/include/meta_data/text_file_meta_data_reader.h

```diff
@@ -44,5 +44,5 @@ class TextFileMetaDataReader : public MetaDataReader {
     bool exists(const std::string& image_name) override;
     void add(std::string image_name, int label);
     std::map<std::string, std::shared_ptr<MetaData>> _map_content;
-    std::string _path;
+    std::string _path, _file_list_path;
```

Comment:
Remove this _file_list_path and use _path itself

Reply:
_path means root path and file_list_path points to the location of file list

Reply:
But in text metadata reader, there is no use for _path variable